UNRELEASED.md

```diff
@@ -14,6 +14,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Adjusted padding to `Card`'s sections, header, and footer to separate actions in header and footer actions in footer ([#962](https://github.com/Shopify/polaris-react/pull/962))
```

Comment:
This could be clearer. Right now I understand that padding changed in several places in Card, but not getting what really changed and why.